cluster/common.sh

```diff
@@ -18,7 +18,7 @@ function install_cdi {
 function install_kubevirt {
     export KUBEVIRT_VER=$(curl -s https://github.com/kubevirt/kubevirt/releases/latest | grep -o "v[0-9]\.[0-9]*\.[0-9]*")
     ./cluster/kubectl.sh apply -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VER}/kubevirt-operator.yaml
-    ./cluster/kubectl.sh create configmap -n kubevirt kubevirt-config --from-literal=feature-gates=DataVolumes
+    ./cluster/kubectl.sh create configmap -n kubevirt kubevirt-config --from-literal=feature-gates=DataVolumes,ImportWithoutTemplate
```

Comment:
Not sure if it's set in stone, but maybe we can also consider to decide whether to import based on if the template resource exists, so we will always fail on openshift, and we will always continue on k8s. And we don't need any config option.

Reply:
With this approach we won't be able to implement functional test for the Openshift behaviour when the template is missing (failing imports). Also, turning this feature off in Openshift would be more difficult and sometimes impossible - someone might want to be able to import VM without a template and use templates for other VMs.

Reply:
I like the approach that a feature is disabled by default and should be enabled explicitly for users that decided to use it.
I see a value in documenting this flag and how to enable it in the design document.

Reply:
:+1: I've just pushed the change in the documentation.